src/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Nullable>enable</Nullable>
     <BaseOutputPath>..\out</BaseOutputPath>
-    <Version>1.1</Version>
+    <Version>1.2</Version>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
The PR description still says: there was a change made in SqlMutationEngine.cs but looking at the original PR, those changed were removed. Looks like the original PR didnt update the description, lets at least update in this cherry-pick PR. 
Also, do we have a similar issue for SQL? If yes, we should fix that too in this version 1.2 (in a separate PR)

Reply:
1. Updated this PR's description to highlight the fact no changes to sqlmutationengine are included.
2. The fix from the original PR affects "typehelper.cs" which is universal. I added a work item to add mirroring tests for SQL: #2319